### PR TITLE
Palaniappan S - 2nd Year CSE B 180501105

### PR DESCRIPTION
**Pull request by PalaniappanSubramanian**

### CODE REVIEW

File: Assignment 2/180501105/occurences.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from collections import Counter as c
+s=input("Enter a string:")
+a=c(s)
+print(a)
```
Comment (mahavisvanathan) [inline Assignment 2/180501105/occurences.py]: Print the output in the required format.

File: Assignment 2/180501105/robberThief.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+s=int(input("Hello robber! How much did you rob?"))
+if s<5000:
+    b="Loss"
+elif (s>5000 and s<10000):
+    b="Profit is Moderate"
+elif s>20000:
+    b="Profit is Good"
+print(b)
+
```
Comment (mahavisvanathan) [inline Assignment 2/180501105/robberThief.py:9]: Make sure your code covers all equal to conditions also

File: Assignment 2/180501105/attendanceStudents.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+attendance=['Ram-P','Malar-A','Ahuja-P','Vijay-P','Harini-A']
+#print("Students present")
+for a in attendance:
+    if a[-1]=='P':
+        print(a)
+
```
Comment (mahavisvanathan) [inline Assignment 2/180501105/attendanceStudents.py:19]: The program must be more dynamic to accept any set of names and their attendance. Feel free to include input lines that will help you make this more dynamic ( for example reading the number of students ). Also, we want to just print the names of the students so find a way to do that.

File: Assignment 2/180501105/occurences.py (hunk 1 above)
Comment (PalaniappanSubramanian) [reply Assignment 2/180501105/occurences.py]: Changes made

File: Assignment 2/180501105/robberThief.py (hunk 2 above)
Comment (PalaniappanSubramanian) [reply Assignment 2/180501105/robberThief.py:9]: changes made

File: Assignment 2/180501105/attendanceStudents.py (hunk 3 above)
Comment (PalaniappanSubramanian) [reply Assignment 2/180501105/attendanceStudents.py:19]: changes made

File: Assignment 2/180501105/attendanceStudents.py [hunk 4]
```diff
@@ -1,6 +1,15 @@
...
+#   if a[-1]=='P':
+#       print(a)
+a=int(input('Enter the No of Students : '))
+h=[]
+print('Enter your string : ')
+for i in range(a):
+	b=input().split('-')
+	if b[1] == 'P':
+		h.append(b[0])
+print('Students Present: '+str(h))
```
Comment (mahavisvanathan) [inline Assignment 2/180501105/attendanceStudents.py]: str(list)  gives the output as 
```
['string1', 'string2', 'string3']
```
change it to the required output 
```
string1 string2 string3
```

**Pull request merged**